cMake/FreeCAD_Helpers/SetGlobalCompilerAndLinkerSettings.cmake

```diff
@@ -42,6 +42,13 @@ macro(SetGlobalCompilerAndLinkerSettings)
                 set (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /EHa")
             endif()
         endif(FREECAD_RELEASE_SEH)
+        if(FREECAD_USE_CCACHE)
```

Comment:
maybe this should be `if(CCACHE_PROGRAM)` instead? `FREECAD_USE_CCACHE` is true by default but it might very well be the case that ccache is not installed in which case these options are unnecessary

Reply:
also I think there are other sutff required for ccache on windows:
https://github.com/FreeCAD/FreeCAD/blob/8f17842974a1c7fba4aaa0e30703690e1c18ffe7/.github/workflows/sub_buildWindows.yml#L127

https://github.com/ccache/ccache/wiki/MS-Visual-Studio#usage-with-cmake

Reply:
For the `conda` builds `ccache` is functioning with `ninja` for builds.  The caching is functioning as intended.

Reply:
@0penBrain do you remember what the other stuff is for?